tool/tsh/mfa.go

```diff
@@ -43,14 +45,17 @@ import (
 	wancli "github.com/gravitational/teleport/lib/auth/webauthncli"
 )
 
+// tshDeviceType represents the device types that appear in the tsh CLI.
+type tshDeviceType = string
```

Comment:
```suggestion
type tshDeviceType string
```

No need for a type alias here.

Reply:
Dropping the alias gets rather painful, as we won't be able to use tshDeviceType directly as a string anymore (which is kind of its reason to be).

Reply:
Then why have the alias at all? You're not getting anything extra out of the type system, so at that point why not just use `string` directly?

Reply:
Fair enough, dropped.